Source/WebCore/svg/properties/SVGAnimatedProperty.h

```diff
@@ -29,20 +29,22 @@
 #include "SVGPropertyOwner.h"
 #include <wtf/RefCounted.h>
 #include <wtf/WeakHashSet.h>
+#include <wtf/WeakPtr.h>
 #include <wtf/text/WTFString.h>
 
 namespace WebCore {
     
 class SVGElement;
+class WeakPtrImplWithEventTargetData;
 
 class SVGAnimatedProperty : public RefCounted<SVGAnimatedProperty>, public SVGPropertyOwner {
 public:
     virtual ~SVGAnimatedProperty() = default;
     
     // Manage the relationship with the owner.
-    bool isAttached() const { return m_contextElement; }
+    bool isAttached() const { return m_contextElement.get() != nullptr; }
```

Comment:
`return m_contextElement;` or `return !!m_contextElement;` if the former doesn't build.